### DISCUSSION TITLE
Where are the config files located?

### DISCUSSION THREAD

**Discussion by winterkami** [flxzt/rnote#508] [Q&A] [answered]
Hi, I would like to export my current rnote config to another computer, but I can't find the config files anywhere. Is there a way to do that?

**Comment by flxzt** [comment, marked as answer]
If you are using flatpak, you should be able to transfer the configuration by copying `~/.var/app/com.github.flxzt.rnote/config/glib-2.0/settings/keyfile`.

If you have the app installed as a native package, it should be possible to export with:

```bash
dconf dump /com/github/flxzt/rnote/ > rnote.conf
```

and load this file on the new machine with:

```bash
dconf load /com/github/flxzt/rnote/ < rnote.conf
```

**Comment by glibg10b** [comment]
If you're on Windows, you can find the settings at `HKCU\SOFTWARE\GSettings\com\github\flxzt\rnote` in the registry. This unfortunately means the config is platform-specific

**Comment by eaves-dropper** [comment]
@flxzt hey I have install rnote on nixos, but I have no `$HOME/.var` and also I can't find anything inside `/var`. Is the location of config file changed?

**Reply by The-An0nym** [reply]
Try `$HOME/.config/dconf/user`
`user` is a binary file that contains information about specific applications (like rnote) and DEs (e.g. Gnome). 
Use `dconf dump /` to see all values (as text).
